extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -454,18 +454,19 @@ void getGraphqlUiFinalDestination(
 
             String graphQLPath = httpRootPath.adjustPath(graphQLConfig.rootPath);
             String graphQLUiPath = nonApplicationRootPathBuildItem
-                    .adjustPath(httpRootPath.adjustPath(graphQLConfig.ui.rootPath));
+                    .adjustPath(graphQLConfig.ui.rootPath);
```

Comment:
Is it on purpose that you're moving `httpRootPath.adjustPath` here once in favour of doing it later several times?

Reply:
Yea, because `NotFoundPageDisplayableEndpointBuildItem` does this under the covers, so I need a var without this for that.

Reply:
OK